packages/nuxt/src/app/composables/fetch.ts

```diff
@@ -140,6 +140,16 @@ export function useFetch<
     controller?.abort?.()
     controller = typeof AbortController !== 'undefined' ? new AbortController() : {} as AbortController
 
+    /**
+     *  workaround for `timout` not working due to custom abort controller
+     * TODO: remove this when upstream issue is resolved
+     * @see https://github.com/unjs/ofetch/issues/326
+     * @see https://github.com/unjs/ofetch/blob/bb2d72baa5d3f332a2185c20fc04e35d2c3e258d/src/fetch.ts#L152
+     */
+    if(toValue(opts.timeout)){
```

Comment:
I think we can cache `toValue()` result. (also is it normal linter not catching space between `if (`?)

Reply:
we need to fix that (cc: @DamianGlowala doing great work on a separate branch)

Reply:
@pi do you mean like this:
```ts
	const timeout = toValue(opts.timeout)
	if (timeout) {
      setTimeout(() => controller.abort(), timeout)
    }
```

Reply:
I believe so 👍